packages/components/breadcrumbs/src/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,74 @@
+import type { Breadcrumbs } from './breadcrumbs.js';
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html } from 'lit';
+import '../register.js';
+
+type Props = Pick<Breadcrumbs, 'homeUrl' | 'noHome'> & { breadcrumbs: TemplateResult };
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Breadcrumbs',
+  args: {
+    homeUrl: '/',
+    noHome: false
+  },
+  argTypes: {
+    breadcrumbs: {
+      table: {
+        disable: true
+      }
+    }
+  },
+  render: ({ breadcrumbs, homeUrl, noHome }) => html`
+    <sl-breadcrumbs .homeUrl=${homeUrl} .noHome=${noHome}>${breadcrumbs}</sl-breadcrumbs>
+  `
+} satisfies Meta<Props>;
+
+export const Basic: Story = {
+  args: {
+    breadcrumbs: html`
+      <a href="javascript:void(0)">Lorem</a>
+      <a href="javascript:void(0)">Ipsum</a>
+      <span>Dolar</span>
+    `
+  }
+};
+
+export const Collapse: Story = {
+  args: {
+    breadcrumbs: html`
+      <a href="javascript:void(0)">Lorem</a>
+      <a href="javascript:void(0)">Ipsum</a>
+      <a href="javascript:void(0)">Dolar</a>
```

Comment:
Hmm I'm not sure if it looks very helpful on smaller devices: 
<img width="409" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/007c1b63-ee51-4fa9-a4b9-b2528e0c8ad2">

 maybe we could have similar behaviour to this one? :)
https://canvas.workday.com/components/navigation/breadcrumbs/#tab=web&heading=overflow-breadcrumbs

Reply:
Fixed